### ISSUE TITLE
Web UI: Unable to view/delete a document with more than 22 properties.

### ISSUE THREAD

**Issue by sm0g** [arangodb/arangodb#1972]
## Environment

- ArangoDB 3.0.3
- OS: OS X El Captain 10.11.6

## Web Browser

Google Chrome 51.0.2704.103 (64-bit)

## Steps to reproduce

1) Open the browser on http://127.0.0.1:8529
2) Log in as root
3) Go to collections tab
3) Create a collection (Document type) with a name "Test"
4) Go into "Test" collection and click green plus button to create a document, click create.
5) In the newly created document switch to code view and paste this sample:

```
{
    "prop_1": "val_1",
    "prop_2": "val_2",
    "prop_3": "val_3",
    "prop_4": "val_4",
    "prop_5": 1,
    "prop_6": 3,
    "prop_7": "val_7",
    "prop_8": "val_8",
    "prop_9": "val_9",
    "prop_10": "val_10",
    "prop_11": "val_11",
    "prop_12": "val_12",
    "prop_13": "val_13",
    "prop_14": "val_14",
    "prop_15": "val_15",
    "prop_16": "val_16",
    "prop_17": "val_17",
    "prop_18": "val_18",
    "prop_19": "val_19",
    "prop_20": "val_20",
    "prop_21": false,
    "prop_22": "val_22",
        
    "prop_23": "val_23"
}
```
6) Click save and go back to "Test" collection view.

As a result a document contains pasted data but _key column is empty. If you click on this document nothing will happen. It's impossible to view and delete it via UI.

![image](https://cloud.githubusercontent.com/assets/518500/17270171/33b750c6-5665-11e6-93d5-98d944f36615.png)

**Comment by CoDEmanX** [comment]
Probably related to #1949. v3.0.4 should fix it and will be released soon, please test when it's out.

**Comment by sm0g** [comment]
Thanks for the answer. I'll definitely check it out.

**Comment by jsteemann** [comment]
Fixed in 3.0.4, which was released today.
